source/docs/installation.md

```diff
@@ -79,5 +75,21 @@ At the time of writing this guide, the output looked like this:
 
 <pre><code class="shell">3.2.1</code></pre>
 
+## Installing Embark @next
+
+While running `npm install -g embark` will give us the latest and great stable version of Embark, we're sometimes interested in trying out features that are in active development. The Embark maintains a `next` distribution tag on npm that can be used to install versions of Embark that aren't stable.
```

Comment:
"The Embark", did you mean "The Embark team"?

Reply:
Yes good catch!